app/views/unattended/report_templates/host_-_last_checkin.erb

```diff
@@ -0,0 +1,28 @@
+<%#
+name: Host - Last Checkin
+snippet: false
+template_inputs:
+- name: Search
+  required: false
+  input_type: user
+  description: A time search after which the Host is considered inactive. It can be
+    any search syntax, e.g. "last_checking < "4 hours ago" and os = RedHat "
+  advanced: false
+  value_type: search
+  resource_type: Host
+  default: last_checkin < "5 hours ago"
+model: ReportTemplate
+require:
+- plugin: katello
+￼ version: 3.16.0
+%>
+<%- load_hosts(search: "#{input('Search')}", preload: [ :subscription_facet, :interfaces ]).each_record do |host| -%>
+<%- report_headers('Name', 'IP', 'Last Checkin') -%>
```

Comment:
would it be worth to use order to have the longer inactive hosts first?

Reply:
the order is a bit problematic, sometimes the result can be nil which we can't order, until we have a solution for that, I wouldn't add it to the default template